### PR TITLE
e2e-openstack-upi: Add 4.5 Installer presubmit

### PR DESCRIPTION
**Pull request by pierreprinetti**
/cc @mandre @adduarte

### CODE REVIEW

File: ci-operator/jobs/openshift/installer/openshift-installer-release-4.5-presubmits.yaml [hunk 1]
```diff
@@ -862,6 +862,92 @@ presubmits:
...
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --kubeconfig=/etc/apici/kubeconfig
+        - --lease-server-password-file=/etc/boskos/password
+        - --lease-server-username=ci
+        - --lease-server=https://boskos-ci.svc.ci.openshift.org
```
Comment (mandre) [inline ci-operator/jobs/openshift/installer/openshift-installer-release-4.5-presubmits.yaml:889]: Please remove, cf https://github.com/openshift/release/pull/11526

**Comment by mandre** [comment]
The job is failing but at least it's running, which is the point of this PR.
/lgtm
/approve

**Comment by pierreprinetti** [comment]
Not interesting any more.

**Pull request closed without merging**